dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
But we don't seem to have sourcemaps setup with any of our CSS files... or else we would see each source file when debugging via browser?

![tree](https://cloud.githubusercontent.com/assets/4960236/5856512/82f2d968-a24c-11e4-9073-a2b66469a133.png)

All I'm seeing is the combined + minified files? (I could be wrong, hence I'm asking)